### ISSUE TITLE
2 junk bands in exported geotiff

### ISSUE THREAD

**Issue by mikelmaron** [timwaters/mapwarper#99]
Uploaded an [image to mapwarper](http://mapwarper.net/maps/15936), and there are issues with the exported GeoTIFF.
- original nodata value was something like -32768
- would indicate it is not uint8
- 5 bands w/ the first colorinterp as grey, and the others as unknown means is also a prob

we are uploading the resulting geotiff to mapbox studio. it need to upload a 3 (RGB) or 4 (RGBA) band geotiff w/ 8 bit pixels. the image displays fine in QGIS because  when u add a raster of ﻿⁠⁠⁠⁠`n﻿`⁠⁠⁠⁠ bands, qgis takes the first three and displays them as rgb
### temporary fix

so basically: bands 4 and 5 were junk. discarded them w/:

`rio calc "(asarray (take a 1) (take a 2) (take a 3))" --co compress=lzw --co tiled=true --co blockxsize=256 --co blockysize=256 --name a=/Users/dnomadb/Downloads/15936.tif /tmp/hi.tif`

then (to be sure) i set the nodata to 0:

`rio edit-info --nodata 0 /tmp/hi.tif`
### fix

ideally mapwarper handles this problem automatically. i'm located the original image and will attach here for debugging

**Comment by mikelmaron** [comment]
Here's the original image https://drive.google.com/file/d/0BzqUHzqXPocbY3hpNm1vQVM4bW8/view

**Comment by timwaters** [comment]
This could be related to #80 and a fix may be in the wikimaps_new branch

gdalinfo NYC1776-mod.png 
Driver: PNG/Portable Network Graphics
Files: NYC1776-mod.png
Size is 1254, 966
Coordinate System is `'
Metadata:
  Raw_profile_type_APP1=
generic profile
      34
49492a0008000000010031010200070000001a000000000000005069636173610000

Image Structure Metadata:
  INTERLEAVE=PIXEL
Corner Coordinates:
Upper Left  (    0.0,    0.0)
Lower Left  (    0.0,  966.0)
Upper Right ( 1254.0,    0.0)
Lower Right ( 1254.0,  966.0)
Center      (  627.0,  483.0)
Band 1 Block=1254x1 Type=Byte, ColorInterp=Red
  Mask Flags: PER_DATASET ALPHA 
Band 2 Block=1254x1 Type=Byte, ColorInterp=Green
  Mask Flags: PER_DATASET ALPHA 
Band 3 Block=1254x1 Type=Byte, ColorInterp=Blue
  Mask Flags: PER_DATASET ALPHA 
Band 4 Block=1254x1 Type=Byte, ColorInterp=Alpha

**Comment by timwaters** [comment]
this is fixed on master now, however there will be a lag for deployment to mapwarper.net

**Comment by timwaters** [comment]
closing because due to deploy
